### DISCUSSION TITLE
Provide AppImage from Ubuntu 22.04

### DISCUSSION THREAD

**Discussion by Nerixyz** [Chatterino/chatterino2#4687] [Ideas]
### Checklist

- [X] I'm reporting a problem with Chatterino
- [X] I've verified that I'm running **the most** recent nightly build or stable release
- [X] I've looked for my problem on the [wiki](https://wiki.chatterino.com/Help/)
- [X] I've searched the [issues and pull requests](https://github.com/Chatterino/chatterino2/issues?q=) for similar looking reports

### Describe your issue

The currently provided AppImage is from Ubuntu 20.04 with Qt 5.12.12 (#4400). This Qt version is EOL and contains bugs visible in Chatterino today (e.g. trails in animated emotes). An AppImage that is built against Qt 5.15 would be really nice (especially for the 7TV fork:  https://github.com/SevenTV/chatterino7/issues/215). 

If I understand AppImages correctly, then once this is provided, support for Qt 5.12 can be dropped since AppImages bundle the Qt dependency (https://github.com/Chatterino/chatterino2/discussions/4292) and users on Ubuntu 20.04 can use the AppImage.

### OS and Chatterino Version

-

**Comment by pajlada** [comment]
We originally built AppImages on 22.04, but it made 20.04 users unable to use the appimage entirely. There's probably a way to make this work, but AppImages are extremely inconvenient to develop for.

Alternatives what work right now for Ubuntu 20.04 or Ubuntu 22.04 users:
1. Use the native .deb files
2. Use the flatpak build

Ways forward, in order of my preference:
1. Host our own PPA that CI updates
2. Host our own Snap version that CI updates
3. Improve the current AppImage
